Client/Client.Qt/Widgets/ChatHistory.cpp

```diff
@@ -62,7 +61,7 @@ void ChatHistory::addMessage(const Network::MessageInfo& messageInfo)
     {
         if (copy->reactions != messageInfo.reactions)
         {
-            auto widgetCopy = std::find_if(_messageList.begin(), _messageList.end(), [messageInfo](const std::unique_ptr<MessageWidget>& mw)
+            auto widgetCopy = std::find_if(_messageList.begin(), _messageList.end(), [messageInfo](const std::shared_ptr<MessageWidget>& mw)
```

Comment:
MessageList field is unique_ptr, what's the reason to change it on shared even if it's const? Again, if this is sonar code smell I don't mind